client/src/state/reducers/selectedExercisesReducer.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash'
+
+//state is not application state, only the state this reducer is responsible for
+export default function(state = [], action){
+    switch(action.type){
+        case 'SELECT_EXERCISE':
+            //return state.concat([action.payload])
+            return [ action.payload, ...state ]
```

Comment:
it's good practice to make action payloads plain objects to start with, so if you have to add things to them, you don't have to change code elsewhere. i.e. your payload could be something like `{ exerciseName: 'kettleBells'}` and within your reducer case you'd have `action.payload.exerciseName`, which is also easier to look at and see what the business logic is.